spec/requests/providers/addresses_request_spec.rb

```diff
@@ -34,7 +34,7 @@
       get_request
 
       expect(response).to be_successful
-      expect(response.body).to include("Enter your client's address manually")
+      expect(response.body).to include(CGI.escapeHTML("Enter your client's address manually"))
```

Comment:
💭 [There's already an helper method](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/master/spec/support/request_helpers.rb#L22-L24) which can be used for this purpose, like in [this example](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/2e1471215a2e3abc3900dfe33a00484aff35fe79/spec/requests/providers/address_selections_spec.rb#L59).

Reply:
I've updated the code here.
I hope I've understood this correctly.